Gemfile

```diff
@@ -44,7 +44,8 @@ gem 'open_id_authentication'
 gem 'i18n'
 
 # Apisonator client
-gem 'pisoni', '~> 1.30'
+# gem 'pisoni', '~> 1.30'
+gem 'pisoni', github: '3scale/pisoni', branch: 'fix-dependencies'
```

Comment:
Let's not put this in master. Just for running the test suite from the PR, that's alright.